Auth0Tests/CredentialsManagerSpec.swift

```diff
@@ -162,6 +162,17 @@ class CredentialsManagerSpec: QuickSpec {
                     }
                 }
             }
+            
+            it("should include custom headers") {
+                stub(condition: hasHeader("foo", value: "bar")) { _ in return revokeTokenResponse() }.name = "revoke success"
+                waitUntil(timeout: Timeout) { done in
+                    credentialsManager.revoke({ error in
+                        expect(error).to(beNil())
+                        expect(credentialsManager.hasValid()).to(beFalse())
```

Comment:
Is this assert necessary for this test case?

Reply:
not really, will remove 👍